modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/documents/group/fonts/sourceSerifPro/source-code-pro-v21-vietnamese_latin_cyrillic-200.eot.license

```diff
@@ -0,0 +1,4 @@
+SPDX-FileCopyrightText: © 2010 - 2020 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font Name ‘Source’.
```

Comment:
@jotchon  Can you squash this with the other font commit so the commits are cleaner? Also the commit message should be ISSITES-5751

Reply:
yes i can totally do that.

Reply:
Fixed with #8